src/parse/run_yaml.jl

```diff
@@ -0,0 +1,171 @@
+"""
```

Comment:
**Organization** This file is related to running yaml files. If these are stored in csv/xlsx columns, it will first write the necessary files.